frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.tsx

```diff
@@ -68,26 +68,36 @@ const ParameterLinkedFilters = ({
     [],
   );
 
+  let disableReason = null;
+  if (usableParameters.length === 0) {
+    disableReason = (
+      <div>
+        <SectionMessage>
+          {t`If you have another dashboard filter, you can limit the choices that are listed for this filter based on the selection of the other one.`}
+        </SectionMessage>
+        <SectionMessage>
+          {jt`So first, ${(
+            <SectionMessageLink key="link" onClick={onShowAddParameterPopover}>
+              {t`add another dashboard filter`}
+            </SectionMessageLink>
+          )}.`}
+        </SectionMessage>
+      </div>
+    );
+  } else if (parameter.values_source_type != null) {
+    disableReason = (
+      <div>
+        <SectionMessage>
+          {t`If the filter has values that are from another question or model, or a custom list, then this filter can't be limited by another dashboard filter.`}
```

Comment:
This text is TBD [here](https://metaboat.slack.com/archives/C057T1QTB3L/p1696844868257629?thread_ts=1696518563.953549&cid=C057T1QTB3L)